ios/CovidShield.xcodeproj/project.pbxproj

```diff
@@ -571,7 +571,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = ca.gc.hcsc.canada.stopcovid;
 				PRODUCT_NAME = CovidShield;
-				PROVISIONING_PROFILE_SPECIFIER = "Stop COVID Development Profile";
+				PROVISIONING_PROFILE_SPECIFIER = "Stop COVID Development";
```

Comment:
Please revert this

Reply:
Whoops, don't know how that snuck in there. Reverted.